app/src/main/java/com/example/plantsapp/presentation/ui/plantdetail/PlantDetailFragment.kt

```diff
@@ -34,6 +34,11 @@ class PlantDetailFragment : Fragment(R.layout.fragment_plant_detail) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        activity?.title = getString(
+            R.string.title_plant_detail_screen,
+            requireArguments().plantName.value
+        )
+
```

Comment:
How about set it from viewmodel? 
Like - you have plant.observe{ } - you can move inside of it

Reply:
> How about set it from viewmodel?
> Like - you have plant.observe{ } - you can move inside of it

good idea, but it takes a second to load plant and user can see how the title on app bar is changing after the screen is loaded

Reply:
> > How about set it from viewmodel?
> > Like - you have plant.observe{ } - you can move inside of it
> 
> good idea, but it takes a second to load plant and user can see how the title on app bar is changing after the screen is loaded

https://user-images.githubusercontent.com/43578259/147236615-49de8c9d-522b-4c49-8f40-48fca6bffeb9.mov

Reply:
Okay - then, maybe add live data for toolbar title? (Only in this place)
It is needed just to have variation data to depend on viewmodel, and not just on arguments